contribute.md

```diff
@@ -1,53 +1,81 @@
+{% include picture.html
+   image = "/assets/images/contribute/contribute.jpg"
+   retina = "/assets/images/contribute/contribute@2x.jpg"
+   mobile = "/assets/images/contribute/contribute-mobile.jpg"
+   mobileRetina = "/assets/images/contribute/contribute-mobile@2x.jpg"
+   alt-text = "Pencils and markers"
+   width = 1600
+   height = 900
+   layout = "full-width"
+%}
+
+# Contribute
+
+**So you’re interested in contributing? Fantastic! We welcome, appreciate and encourage new contributions.**
+
+Depending on your skills and interest, you can help in a number of ways. We design, illustrate, write copy and code, do technical research, discuss and answer questions, run workshops and community calls, and more. Design is a multidisciplinary activity and there are many different ways to be part of this community and help improve how we interact with the Bitcoin protocol.
+
+
+## Learn how we work
+
+If you are not familiar with the concepts of open design, we have a [great intro]({{ '/guide/getting-started/open-design/' | relative_url }}) for you. Open, public collaboration gives a lot of freedom to each participant and relies on us organically coming up with the organizational structures we want. The result is that things can sometimes look unorganized or unintuitive from the outside. The big benefit is that you can choose how you want to participate. Let's jump in.
+
+## How to get involved in the community
+
+1. [Join the community on Slack](https://join.slack.com/t/bitcoindesign/shared_invite/zt-gytq2snl-4TEWJOTKrXRCB4YLBoDunA)
+2. [Subscribe to the newsletter](https://bitcoindesign.substack.com) to stay up-to-date
+2. Read up on our [project life cycle](Projects.md)
+3. Browse [issues](https://github.com/BitcoinDesign/Meta/issues) for upcoming calls and discussions around processes and coordination
+4. Subscribe to our [calendar](https://raw.githubusercontent.com/BitcoinDesign/Meta/calendar/events.ical) for calls and events we organize
+
+## How to get involved in projects
+
+1. See the projects we are either involved in or recommend on the [projects]({{ 'projects' | relative_url }}) page
+1. Browse [issues labelled as projects](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aopen+is%3Aissue+label%3Aproject)
+2. Browse and post ideas in the [#project-ideas](https://bitcoindesign.slack.com/archives/C0174N5KUF9) Slack channel
+1. See [projects](Projects.md) for a few project ideas
+1. Contribute to the [Bitcoin Design Guide](https://github.com/BitcoinDesign/Guide)
+
+## Bitcoin Design Guide
+
+- Project Intro — [Doc v2.0.0](https://docs.google.com/document/d/1YiYeRIybGmxmErCOI4Jc8Qajz3JGM1JYVfUtpzyCzSk/edit?usp=sharing)
+- Slack Channel — [#bitcoin-design-guide](https://bitcoindesign.slack.com/archives/C015856BDME)
+- Github Repository — [Guide](https://github.com/BitcoinDesign/Guide)
+- Github Project Board — [Guide/Content](https://github.com/BitcoinDesign/Guide/projects/1)
+
```

Comment:
```suggestion
```suggestion
[How To Contribute to Bitcoin Design Guide video series](https://www.youtube.com/playlist?list=PLPZzDjXV0FTZwILz4uyqiJNjUX-kHWP0k)

```

```